.travis.yml

```diff
@@ -94,7 +94,8 @@ install:
 
 before_script:
   - travis_retry ./testing/downloadSpark.sh $SPARK_VER $HADOOP_VER
-  - ./testing/setupLivy.sh
+  - ./testing/setupLivy.sh $LIVY_VER
+  - if [[ -n $LIVY_VER ]]; then export LIVY_HOME=`pwd`/livy-server-$LIVY_VER; fi
```

Comment:
It seems export `LIVY_HOME` in `setupLivy.sh` doesn't work, I have to export it in `.travis.yml`